infra/concourse/build/developer-tools/Dockerfile

```diff
@@ -0,0 +1,78 @@
+ARG RUBY_VERSION
+FROM ruby:$RUBY_VERSION-alpine
+
+# Required to download and install Terraform
+ARG TERRAFORM_VERSION
+ENV TERRAFORM_VERSION ${TERRAFORM_VERSION}
+
+# Required to download and install terraform-docs
+ARG TERRAFORM_DOCS_VERSION
+ENV TERRAFORM_DOCS_VERSION ${TERRAFORM_DOCS_VERSION}
+
+# Required to download and install the Terraform gsuite provider
+ARG GSUITE_PROVIDER_VERSION
+ENV GSUITE_PROVIDER_VERSION ${GSUITE_PROVIDER_VERSION}
+
+# Required to download and install Google Cloud SDK
+# Google Cloud SDK is located at /usr/local/google-cloud-sdk
+ARG CLOUD_SDK_VERSION
+ENV CLOUD_SDK_VERSION ${CLOUD_SDK_VERSION}
+
+# $WORKSPACE is intended for assets that persist across multiple build steps in a pipeline.
+# It's also where the project git repository is located.
+# https://cloud.google.com/cloud-build/docs/build-config
+ENV WORKSPACE="/workspace"
+ENV PATH /usr/local/google-cloud-sdk/bin:$PATH
+
+# bash is preferred over /bin/sh
+RUN apk add --no-cache bash
+
+# All package-level dependencies are handled by install_dependencies.sh
+ADD ./build/install_dependencies.sh /build/
+RUN /build/install_dependencies.sh
+
+ADD ./build/install_cloud_sdk.sh /build/
+RUN /build/install_cloud_sdk.sh ${CLOUD_SDK_VERSION}
+
+ADD ./build/install_gsuite_terraform_provider.sh /build/
+RUN /build/install_gsuite_terraform_provider.sh ${GSUITE_PROVIDER_VERSION}
+
+ADD ./build/install_terraform.sh /build/
+RUN /build/install_terraform.sh ${TERRAFORM_VERSION}
+
+ADD ./build/install_shellcheck.sh /build/
+RUN /build/install_shellcheck.sh
+
+ADD ./build/install_hadolint.sh /build/
+RUN /build/install_hadolint.sh
+
+ADD ./build/install_terraform_docs.sh /build/
+RUN /build/install_terraform_docs.sh ${TERRAFORM_DOCS_VERSION}
+
+ADD ./build/install_kubectl.sh /build/
+RUN /build/install_kubectl.sh
+
+ADD ./build/scripts/setup_environment.sh /usr/local/bin/
+ADD ./build/scripts/execute_with_credentials.sh /usr/local/bin/
+
+# Provide task helper functions with wrapper scripts for Terraform module lint
+# checks
+ADD ./build/scripts/task_helper_functions.sh /usr/local/bin/
+ADD ./build/scripts/task_wrapper_scripts/* /usr/local/bin/
+
+WORKDIR /opt/kitchen
+ADD ./build/data/Gemfile .
+ADD ./build/data/Gemfile.lock .
+ADD ./build/data/requirements.txt .
+RUN bundle install && pip3 install -r requirements.txt
+
+WORKDIR $WORKSPACE
+RUN terraform --version && \
+    terraform-docs --version && \
+    gcloud --version && \
+    ruby --version && \
+    bundle --version && \
+    kubectl version --client=true
+
+# Cleanup intermediate build artifacts
+RUN rm -rf /build
```

Comment:
Suggest overriding the entrypoint or CMD or whatever is causing this behavior:

```
docker run --rm -it cft/developer-tools:0.0.1
irb(main):001:0>
```

I prefer dropping to an interactive bash shell since this is how many of us work with the developer tools image based on the quick survey we took in our video call.

Reply:
Addressed in 5e2901e